data/core/units/undead/Necromancer.cfg

```diff
@@ -54,9 +51,10 @@ This ability, in all aspects, is the first step towards cheating death of its ul
         type=arcane
         [specials]
             {WEAPON_SPECIAL_MAGICAL}
+            {WEAPON_SPECIAL_PLAGUE}
         [/specials]
         range=ranged
-        damage=16
+        damage=12
```

Comment:
maybe 14-2? thr 16 -> 12 seems a bit harsh. Nercomancer is a....common enemy leader unit in most UMC/scenarios, so nerfing to this state lowers the difficulty curve by a lot. Not sure of the state of SotA...but I guess it will be way harder now (as you dont get access to Lich until the story progresses to that phase.) DiD also affected as Darken Volk is easily dealt with now. His arcane range made me cautious as even Wraiths cannot tank out that attack. Now, I can rush these poor sod with Deathblades.

Reply:
> maybe 14-2? thr 16 -> 12 seems a bit harsh. Nercomancer is a....common enemy leader unit in most UMC/scenarios, so nerfing to this state lowers the difficulty curve by a lot.

Shadow Wave is the Necromancer's secondary arcane attack - I wouldn't expect this to be used by the AI against the player unless the player is also undead. Regarding SotA, my goal is to make the Necro's overall strength similar to it's old 1.18 strength, just focused more around Plague than raw damage output.

That said, Hejenewar also feels the new Necro is dramatically weaker than the old Necro, so I'm thinking of buffing its Chill Wave cold attack back up to retain its damage output.

> DiD also affected as Darken Volk is easily dealt with now. His arcane range made me cautious as even Wraiths cannot tank out that attack.

12x2 was his old damage value from 1.16, so I don't think it's an unworkable change. But I agree that if DiD was changed in 1.18 to compensate for 16x2, those elements might need to be reverted back to compensate for 12x2.

Reply:
> 12x2 was his old damage value from 1.16. Oh, in that case. the new changes are all fine.
and no, SotA and DiD have not had balance runs for quite a while.